app/src/main/java/org/asteroidos/sync/MainActivity.java

```diff
@@ -35,39 +32,72 @@
         DeviceListFragment.OnScanRequestedListener, DeviceDetailFragment.OnDefaultDeviceUnselectedListener,
-        DeviceDetailFragment.OnConnectRequestedListener, BleManager.DiscoveryListener,
-        DeviceDetailFragment.OnAppSettingsClickedListener, DeviceDetailFragment.OnLocationSettingsClickedListener,
-        DeviceDetailFragment.OnUpdateListener {
-    private BleManager mBleMngr;
-    private DeviceListFragment mListFragment;
-    private DeviceDetailFragment mDetailFragment;
-    private Fragment mPreviousFragment;
-    Messenger mSyncServiceMessenger;
-    Intent mSyncServiceIntent;
-    final Messenger mDeviceDetailMessenger = new Messenger(new MainActivity.SynchronizationHandler(this));
-    int mStatus = SynchronizationService.STATUS_DISCONNECTED;
-
-    public static ArrayList<AppInfo> appInfoList;
+        DeviceDetailFragment.OnConnectRequestedListener, DeviceDetailFragment.OnAppSettingsClickedListener,
+        DeviceDetailFragment.OnLocationSettingsClickedListener, DeviceDetailFragment.OnUpdateListener {
 
     public static final String PREFS_NAME = "MainPreferences";
     public static final String PREFS_DEFAULT_MAC_ADDR = "defaultMacAddress";
     public static final String PREFS_DEFAULT_LOC_NAME = "defaultLocalName";
+    private static final String TAG = "MainActivity";
+    public static ArrayList<AppInfo> appInfoList;
+    final Messenger mDeviceDetailMessenger = new Messenger(new MainActivity.SynchronizationHandler(this));
+    public ParcelUuid asteroidUUID = fromString(AsteroidUUIDS.SERVICE_UUID.toString());
+    Messenger mSyncServiceMessenger;
+    /* Synchronization service events handling */
+    private final ServiceConnection mConnection = new ServiceConnection() {
+        public void onServiceConnected(ComponentName className,
+                                       IBinder service) {
+            mSyncServiceMessenger = new Messenger(service);
+            onUpdateRequested();
+        }
+
+        public void onServiceDisconnected(ComponentName className) {
+            mSyncServiceMessenger = null;
+        }
+    };
+    Intent mSyncServiceIntent;
+    int mStatus = SynchronizationService.STATUS_DISCONNECTED;
+    ScanSettings settings;
+    List<ScanFilter> filters;
+    private DeviceListFragment mListFragment;
+    public ScanCallback scanCallback = new ScanCallback() {
+        @Override
+        public void onScanResult(int callbackType, @NonNull ScanResult result) {
+            super.onScanResult(callbackType, result);
 
+            if (mListFragment == null) return;
+            mListFragment.deviceDiscovered(result.getDevice());
+            Log.d(TAG,"SCAN RESULT:" + result.getDevice().toString() + " Name:" + result.getDevice().getName());
+            ParcelUuid[] arr = result.getDevice().getUuids();
+        }
+    };
+    private DeviceDetailFragment mDetailFragment;
+    private Fragment mPreviousFragment;
+    private BluetoothLeScannerCompat scanner;
```

Comment:
It looks like you did not apply the mMember convention consistently, I still see quite a few members which don't start with a m